benchmarks/quac/quac.py

```diff
@@ -0,0 +1,78 @@
+"""
+QuAC: A Reading Comprehension Benchmark Requiring Discrete Reasoning Over Dialogs
+
+Eunsol Choi, He He, Mohit Iyyer, Mark Yatskar, Wen-tau Yih†, Yejin Choi, Percy Liang, Luke Zettlemoyer
+https://arxiv.org/pdf/1808.07036
+
+# eval all subjects w/ 500 randomly selected samples
+inspect eval quac.py --limit 1000
+"""
+
+from f1 import f1_scorer
+
+from inspect_ai import Task, task
+from inspect_ai.dataset import Sample
+from inspect_ai.dataset._sources.hf import hf_dataset
+from inspect_ai.model._generate_config import GenerateConfig
+
+
+@task
+def quac():
+    dataset = hf_dataset(
+        path="allenai/quac",
+        sample_fields=record_to_sample,
+        split="validation",
+        shuffle=True,
+        trust=True,
+    )
+    return Task(
+        dataset=dataset,
+        scorer=f1_scorer(),
+        config=GenerateConfig(temperature=0.5),
```

Comment:
set to 0 to make deterministic, as done for commonsense_qa benchmark?

Reply:
I cant see anything in the paper or the alternative implementation repo you linked above about the temperature. Perhaps make it deterministic and also expose it as a param for the user to set?